tsconfig.json

```diff
@@ -18,7 +18,16 @@
   "typedocOptions": {
     "mode": "file",
     "module": "commonjs",
+    "excludeExternals": true,
     "excludePrivate": true,
-    "excludeNotExported": true
+    "excludeNotExported": true,
+    "name": "Unstoppabledomains resolution library",
+    "inputFiles": [
+      "./src/Resolution.ts",
+      "./src/index.ts",
+      "./src/publicTypes.ts",
+      "./src/errors/configurationError.ts",
```

Comment:
@JohnnyJumper please, recheck types and classes inside error files. I'm not sure we need types like `ConfigurationErrorHandler` or `ResolutionErrorHandler` to be exposed to docs.

Reply:
Good catch! I removed them